### PR TITLE
setup.py: Add classifiers and python_requires

### PR DESCRIPTION
**Pull request by vilhelmprytz**
## Description of new feature, or changes

I have added a list of `classifiers` to the `setup.py`. This is mostly metadata but will make it easier to see what versions are supported. Also added `python_requires` which blocks older versions. I have assumed that Halo does not support versions older than 3.4. If this is incorrect it should be fairly easily adjustable.

## Checklist

- [x] Your branch is up-to-date with the base branch
- [x] You've included at least one test if this is a new feature
- [x] All tests are passing

## Related Issues and Discussions

Currently, Halo shows up as "not python 3 compatible" on sites like [pyup.io](https://pyup.io). I am aiming to solve this problem.

## People to notify
N/A

### CODE REVIEW

**Comment by vilhelmprytz** [comment]
Is the Travis check misconfigured here? It hasn't been triggered at all.

**Comment by vilhelmprytz** [comment]
Any news regarding this?

File: setup.py [hunk 1]
```diff
@@ -18,6 +18,16 @@ def dependencies(file):
     packages=find_packages(exclude=("tests", "examples")),
     version="0.0.30",
     license="MIT",
+    classifiers=[
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.4",
```
Comment (manrajgrover) [inline setup.py:23]: We don't test the package on Python 3.4. Are you sure it works for this version?

File: setup.py [hunk 2]
```diff
@@ -18,6 +18,16 @@ def dependencies(file):
     packages=find_packages(exclude=("tests", "examples")),
     version="0.0.30",
```
Comment (manrajgrover) [inline setup.py:19]: Can you increment the version to make a release?

**Review by manrajgrover — CHANGES_REQUESTED** [review]
@vilhelmprytz Thanks for contributing! :)

I've left a few comments.

File: setup.py (hunk 1 above)
Comment (vilhelmprytz) [reply setup.py]: Solved, removed it.

File: setup.py (hunk 2 above)
Comment (vilhelmprytz) [reply setup.py]: Sure thing, done!

**Comment by manrajgrover** [comment]
Thanks @vilhelmprytz !

**Comment by vilhelmprytz** [comment]
@manrajgrover Have you pushed to PyPI yet? You should use GitHub releases so that you can via your CI automatically build and push to PyPI upon release :+1:

**Comment by manrajgrover** [comment]
I've made a release on PyPI.

https://pypi.org/project/halo/0.0.31/

I haven't explored CI for releases. I'll have a look for future releases.

Thanks again!

**Pull request merged**